src/preact/base-element.js

```diff
@@ -480,6 +548,18 @@ PreactBaseElement['props'] = {};
  */
 PreactBaseElement['children'] = null;
 
+/**
+ * @param {typeof PreactBaseElement} Ctor
+ * @return {boolean}
+ */
+function isShadowMapping(Ctor) {
```

Comment:
What is shadow "mapping"? Maybe call this `usesShadowDom`?

Reply:
Done.